src/main/resources/templates/fragments/download-and-preview-home.html

```diff
@@ -4,9 +4,9 @@
     <p class="data-representations">
       <th:block th:if="${totalRecords} le 5000">Download a copy of this register as</th:block>
       <th:block th:if="${totalRecords} gt 5000">Download the first 5,000 records as</th:block>
-      <a href="/records.csv?page-size=5000" rel="alternate" class="js-download" data-download-type="CSV">CSV</a>,
-      <a href="/records.tsv?page-size=5000" rel="alternate" class="js-download" data-download-type="TSV">TSV</a> or as a
-      <a href="/records.xlsx?page-size=5000" rel="alternate" class="js-download" data-download-type="XLSX">spreadsheet</a>.
+      <a href="/records.csv?page-size=5000" rel="alternate" class="js-download" data-download-type="CSV" download="">CSV</a>,
```

Comment:
@danmitchell- can correct me on this but as far as I know, Custom Data Attributes start with `data-` in HTML5.

Reply:
@danmitchell- this is not a data attribute. See https://www.w3schools.com/tags/att_a_download.asp

Reply:
@gidsg I saw that, but I thought it was just `download` without `=""` . I was not sure what it was. Just ignore the comment then

Reply:
Correct. Its not a `data` attribute.

Reply:
Not sure if im missing something here but you should be passing the URL into the download attribute i.e. `<a href="/preview-records/json" download="/preview-records/json">JSON</a>`

Reply:
No, it uses the URL from the href. The value is optional and is used to override the filename given by the server.

Reply:
Ah yes. Not sure how much I trust that but 👌